Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/ruleset/AbstractKeyView.java

```diff
@@ -31,6 +31,11 @@
      */
     protected final U universal;
 
+    /**
+     * The universal rule.
```

Comment:
For what?

Reply:
To get the `minOccurs` and `maxOccurs` value.